README.md

```diff
@@ -287,6 +287,36 @@ Note that this will override any existing `id` prop if specified before calling
 <input {...text('username')} id="signup-username" />
 ```
 
+## Custom Controls
+
+`useFormState` provides a `raw` type for working with controls that do not use React's [`SyntheticEvent`](https://reactjs.org/docs/events.html) system.  For example, controls like [react-select](https://react-select.com/home) or [react-datepicker](https://www.npmjs.com/package/react-datepicker) have `onChange` and `value` props that expect a raw value instead of an event.
+
+To use this, your custom component should support an `onChange()` event which takes the value as a parameter, and a `value` prop which is expected to contain the value.  Note that if no default value is given, the component will receive a `value` prop of `undefined`.
+
+```js
+import DatePicker from 'react-datepicker';
+
+function Widget() {
+  const [formState, { raw }] = userFormState({ date: new Date() });
+
+  return <>
+    <DatePicker {...raw('date')} />
+  </>;
+}
+```
+
+By default, no `onBlur()` prop is passed to the custom component, but if you have a component which supports `onBlur`, you can instruct `useFormState` to pass one.  `onBlur()` will not be passed any value.
```

Comment:
This line isn't very clear to me. When `raw` is used, what do you mean by:

> no `onBlur()` prop is passed [...] but if you have a component which supports `onBlur`, you can instruct `useFormState` to pass one

Reply:
Agreed.  And it's not even accurate anymore.